postgresql-plugin/src/e2e-test/features/postgresql/sink/PostgresqlRunTimeMacro.feature

```diff
@@ -0,0 +1,87 @@
+    Then Validate "PostgreSQL" plugin properties
+    Then Close the Plugin Properties page
+    Then Save the pipeline
+    Then Preview and run the pipeline
+    Then Enter runtime argument value "projectId" for key "bqProjectId"
+    Then Enter runtime argument value "projectId" for key "bqDatasetProjectId"
+    Then Enter runtime argument value "dataset" for key "bqDataset"
+    Then Enter runtime argument value "bqSourceTable" for key "bqTable"
+    Then Enter runtime argument value "driverName" for key "PostgreSQLDriverName"
+    Then Enter runtime argument value from environment variable "host" for key "PostgreSQLHost"
+    Then Enter runtime argument value from environment variable "port" for key "PostgreSQLPort"
+    Then Enter runtime argument value from environment variable "username" for key "PostgreSQLUsername"
+    Then Enter runtime argument value from environment variable "password" for key "PostgreSQLPassword"
+    Then Enter runtime argument value "databaseName" for key "PostgreSQLDatabaseName"
+    Then Enter runtime argument value "targetTable" for key "PostgreSQLTableName"
+    Then Enter runtime argument value "schema" for key "PostgreSQLSchemaName"
+    Then Run the preview of pipeline with runtime arguments
+    Then Wait till pipeline preview is in running state
+    Then Open and capture pipeline preview logs
+    Then Verify the preview run status of pipeline in the logs is "succeeded"
+    Then Close the pipeline logs
+    Then Close the preview
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Enter runtime argument value "projectId" for key "bqProjectId"
+    Then Enter runtime argument value "projectId" for key "bqDatasetProjectId"
+    Then Enter runtime argument value "dataset" for key "bqDataset"
+    Then Enter runtime argument value "bqSourceTable" for key "bqTable"
+    Then Enter runtime argument value "driverName" for key "PostgreSQLDriverName"
+    Then Enter runtime argument value from environment variable "host" for key "PostgreSQLHost"
+    Then Enter runtime argument value from environment variable "port" for key "PostgreSQLPort"
+    Then Enter runtime argument value from environment variable "username" for key "PostgreSQLUsername"
+    Then Enter runtime argument value from environment variable "password" for key "PostgreSQLPassword"
+    Then Enter runtime argument value "databaseName" for key "PostgreSQLDatabaseName"
+    Then Enter runtime argument value "targetTable" for key "PostgreSQLTableName"
+    Then Enter runtime argument value "schema" for key "PostgreSQLSchemaName"
+    Then Run the Pipeline in Runtime with runtime arguments
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Close the pipeline logs
+    Then Validate the values of records transferred to target PostGreSQL table is equal to the values from source BigQuery table
```

Comment:
We haven't covered the Macro scenario for Basic Section. Can we add a test.

Reply:
Added the test.